server/datastore/mysql/migrations/tables/20220608113128_AddTransparencyURLToAppConfig.go

```diff
@@ -0,0 +1,51 @@
+package tables
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220608113128, Down_20220608113128)
+}
+
+func Up_20220608113128(tx *sql.Tx) error {
+	var raw []byte
```

Comment:
This will not be the last time we update `app_config_json`, and we've done it in `server/datastore/mysql/migrations/tables/20220404091216_UpdateAppConfigLogEndpoint.go` too, maybe create a method for it? `updateAppConfigJSON(tx *sql.Tx, fn func(config *fleet.AppConfig)) error`